src/pages/HomePage/components/Questions/Questions.tsx

```diff
@@ -3,7 +3,60 @@ import './Questions.scss';
 export const Questions= () =>{
     return(
         <>
-        <h1>Questions</h1>
+        <div className="QuestionBlog narrow-container">
+        <div className="Blog">
```

Comment:
you must create Question component a the same level as Questions component (no need to add a new "components" folder)
also, you need to create an array for you questions inside of this component and use questions.map(...) to render all your Question components

also I think each question must have an answer. so you need to create some text for this)
and later we will do questions to be expandable